proto/desmos/profiles/v1beta1/models_profile.proto

```diff
@@ -0,0 +1,59 @@
+syntax = "proto3";
+package desmos.profiles.v1beta1;
```

Comment:
Is there any reason to call it v1beta1 instead of v2?

Reply:
We need it to be backward-compatible with the current Proto definitions, which all live inside the `desmos.profiles.v1beta1` namespace. Changing this would result in a different Protobuf URL to be used when registering this type inside the `interfaceRegistry`, which would lead to Desmos not be able to properly read old profiles (and thus crashing while migrating)